net_util/src/tap.rs

```diff
@@ -34,8 +34,10 @@ pub enum Error {
     IoctlError(c_ulong, #[source] IoError),
     #[error("Failed to create a socket")]
     NetUtil(#[source] NetUtilError),
-    #[error("Invalid interface name")]
-    InvalidIfname,
+    #[error("interface name too long (max length is 15): {0}")]
```

Comment:
nit: Missing capital letter?

Reply:
For consistency with the CHV code base, yes. For consistency with the Rust style guide for error messages: no

But I think we should stick to consistency with the code base!